aspnetcore/tutorials/choose-web-ui.md

```diff
@@ -162,7 +162,7 @@ Choose ASP.NET Core MVC plus Blazor WebAssembly when these benefits best fit you
 * Prerendering:
   * Executes Razor components on the server and renders them into a page or view.
       * Improves the perceived load time of the app while setting up interactivity.
-* Add islands of interactivity to existing views (pages) with the [Component Tag Helper](xref:mvc/views/tag-helpers/builtin-th/component-tag-helper).
+* Add islands of interactivity to existing views (pages) with the <xref:mvc/views/tag-helpers/builtin-th/component-tag-helper>.
```

Comment:
This one will throw the title that includes "in ASP.NET Core" such that it won't jive with the text. It will read ...

> Add islands of interactivity to existing views (pages) with the Component Tag Helper in ASP.NET Core.

Reply:
```suggestion
* Add islands of interactivity to existing views (pages) with the [Component Tag Helper](xref:mvc/views/tag-helpers/builtin-th/component-tag-helper).
```